src/components/AccountDetails/index.tsx

```diff
@@ -275,7 +277,9 @@ export default function AccountDetails({
                             href={getExplorerLink(chainId, ENSName, 'address')}
                           >
                             <CustomLinkIcon size={16} />
-                            <span style={{ marginLeft: '4px' }}>View on block explorer</span>
+                            <span style={{ marginLeft: '4px' }}>
+                              {t('viewOn')} {t('blockExplorer')}
```

Comment:
Should we use interpolation here? I think it's better because "View on" always need context!

Reply:
done